src/MeldingsLogg.elm

```diff
@@ -0,0 +1,31 @@
+module MeldingsLogg exposing (MeldigsLoggInfo, MeldingsLogg(..), init, leggTilSpørsmål, leggTilSvar, meldinger)
```

Comment:
Her burde dere ikke expose `MeldingsLogg(..)`, men bare `MeldingsLogg` for å gjøre den opaque. Og `MeldigsLoggInfo` burde dere ikke expose i det hele tatt, for den er bare for bruk internt i modulen.